biom/table.py

```diff
@@ -2722,6 +2723,60 @@ def transform(self, f, axis='sample', inplace=True):
 
         return table
 
+    def rankdata(self, axis='sample', inplace=True, method='average'):
+        """Convert counts to rank abundances from smallest to largest
```

Comment:
It isn't always the case that the data contained are counts as they may be relative abundances.